packages/commerce-sdk-react/src/hooks/utils.test.ts

```diff
@@ -46,3 +46,26 @@ describe('Hook utils', () => {
         })
     })
 })
+
+describe('getCustomerKey', function () {
```

Comment:
"getCustomerKey" ==> "getCustomKey"

Also did you want to add a test for the scenario where it throws and error please?